src/lab4.py

```diff
@@ -0,0 +1,50 @@
+class WaterPump:
+
+    def __init__(self, power_consumption=0, brand="", volume_ph=0, price=0, name=""):
+        self.__power_consumption = power_consumption 
+        self.__brand = brand
+        self.__volume_ph = volume_ph
+        self.price = price
+        self.name = name
+
+
+    
+    def get_power_consumption(self):
+        return self.__power_consumption
+
+    def get_brand(self):
+        return self.__brand
+
+    def get_volume_ph(self):
+        return self.__volume_ph
+
+
+
+    def __str__(self):
+        return f"{self.__power_consumption}, {self.__brand}, {self.__volume_ph}, {self.price}, {self.name}"

```

Comment:
Add type hints for constructor parameters to enhance readability and type safety.